main.js

```diff
@@ -175,7 +186,7 @@ async function writeWrapper(msysRootDir, pathtype, destDir, name) {
 
 async function runMsys(args, opts) {
   assert.ok(cmd);
-  const quotedArgs = args.map((arg) => {return `'${arg.replace(/'/g, `'\\''`)}'`});
+  const quotedArgs = args.map((arg) => {return `'${arg.replace(/'/g, `'\\''`)}'`}); // fix confused vim syntax highlighting with: `
```

Comment:
Is this comment left here on purpose?

Reply:
yes, in a separate commit.  Vim's confusion with that line has been bugging me for a long time, so I added a workaround.

Reply:
I don't really understand the meaning. What is the solution that fixed your issue? The fact that you added a comment ending in a backtick?

Reply:
yes.  Vim thought there was an unterminated backtick string in that line, and the backtick in the comment "closed" that and got the syntax highlighter back in sync.